misk/src/main/kotlin/misk/Action.kt

```diff
@@ -4,8 +4,8 @@ import kotlin.reflect.KFunction
 import kotlin.reflect.KType
 
 data class Action(
-  val name: String,
-  val function: KFunction<*>,
-  val parameterTypes: List<KType>,
-  val returnType: KType
+    val name: String,
```

Comment:
might need to update either intellij or your code style, constructor properties should have 2 space indent (assuming 2 space indent is the norm for the project)

Reply:
Yup. I think I got that wrong. Fixing.

Reply:
Actually hang on; I think the _Square Android_ style guide is the one that’s wrong. Should be double the normal indent for parameters lists.
https://android.github.io/kotlin-guides/style.html#functions